We can't use execerror yet because it assumes the jumpbuf has
been setup which it hasn't just yet.  Dump errors with fprintf
to stderr instead.

Patch by Mike Frysinger <vapier@gentoo.org>

--- pcalc-000/pcalc.y
+++ pcalc-000/pcalc.y
@@ -188,7 +188,7 @@
         yyin = fopen(&argv[1][1], "rt");
         if(!yyin)
             {
-            printf("Cannot find file.\n");
+            fprintf(stderr, "Cannot find file.\n");
             exit(0);
             }
         }
@@ -211,7 +211,7 @@
 
         if(!yyin)
             {
-            execerror( "cannot create tmp file\n", NULL); exit(0);
+            fprintf(stderr, "cannot create tmp file\n"); exit(0);
             }
         fwrite(buff, len, 1, yyin);
         fputc('\n', yyin);
